src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.test.js

```diff
@@ -261,7 +319,7 @@ describe('signals/incident-management/containers/IncidentSplitContainer', () =>
 
     expect(dispatch).toHaveBeenCalledWith(
       showGlobalNotification({
-        title: 'De melding is succesvol gedeeld',
+        title: 'Deelmelding gemaakt',
```

Comment:
Did you discuss this change with the designer? I believe here could be more that one Incident and the existing message was covering that.

Reply:
Good catch, it does not account for multiple messages. Checked with designer and changed it to _Deelmelding(en) gemaakt_